integrations/tensorflow/e2e/slim_vision_models/BUILD

```diff
@@ -102,10 +109,12 @@ iree_slim_vision_test_suite(
         },
     ],
     models = [
+        # "amoebanet_a_n18_f448",  # SavedModelV2 (classification/4) not available.
```

Comment:
Why is this commented out here instead of being marked as failing above?

Reply:
Good point. I'll put it in there and add a filter to exclude it from the coverage table (since it's not interesting that someone hasn't exported this yet).

Reply:
Ah no one has exported the module yet, I see :-D I think that's actually a reasonable reason to have it commented out instead of marked as failing, but either way.